### ISSUE TITLE
TypeError: rasterize_gaussians(): incompatible function arguments. The following argument types are supported:

### ISSUE THREAD

**Issue by Dragon2938734** [ShunyuanZheng/GPS-Gaussian#19]
Is there anybody sattle this bug? In other 3dgs project, i never meet this bug, only in [GPS-Gaussian]. how can i solve it?

TypeError: rasterize_gaussians(): incompatible function arguments. The following argument types are supported:
    1. (arg0: torch.Tensor, arg1: torch.Tensor, arg2: torch.Tensor, arg3: torch.Tensor, arg4: torch.Tensor, arg5: torch.Tensor, arg6: torch.Tensor, arg7: torch.Tensor, arg8: float, arg9: torch.Tensor, arg10: torch.Tensor, arg11: torch.Tensor, arg12: float, arg13: float, arg14: int, arg15: int, arg16: torch.Tensor, arg17: int, arg18: torch.Tensor, arg19: bool, arg20: bool) -> Tuple[int, torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor]

**Comment by LiQiiiii** [comment]
Hi, have you solve this problem? I meet the same problem.

**Comment by Li-can-cheng** [comment]
> s

So am I

**Comment by Li-can-cheng** [comment]
> Hi, have you solve this problem? I meet the same problem.

So am I

**Comment by dharmasai21** [comment]
is the problem solved? what is the soln ?

**Comment by dharmasai21** [comment]
Apparently, This is a problem that comes when there is a mismatch with diff-gaussian-rasterization submodule. one solution is discard the entire environment you might have created, clone the git again, create a new conda environment & start all over.

**Comment by Songxinlei** [comment]
have you solve this problem?

**Comment by dharmasai21** [comment]
Yeah. I got this error when i was executing one of the already existing implementations of another paper. After i reinstalled the conda environment, the error was gone

**Comment by Songxinlei** [comment]
> Yeah. I got this error when i was executing one of the already existing implementations of another paper. After i reinstalled the conda environment, the error was gone. For your case you may try reinstalling just the submodule part.

Thank you for your reply.That is work!
